.github/actions/test/action.yml

```
@@ -0,0 +1,11 @@
+name: run_tests
+description: Run tests
+runs:
+    using: composite
+    steps:
+    - name: run tests
+      run: |
+        for i in {1..3}; do
+          npm test && break
+        done
```

```suggestion
        for i in {1..3}; do
          npm test && break
        done
```
@shayan-deriv, could you please clarify on why the `for` `loop` is necessary? 🤔

---

it doesn't make sense for me either. I checked with the binary-bot team . will remove it